src/thor/bidirectional_astar.cc

```diff
@@ -206,8 +206,8 @@ inline bool BidirectionalAStar::ExpandInner(baldr::GraphReader& graphreader,
   const uint64_t localtime = time_info.valid ? time_info.local_time : 0;
   uint8_t restriction_idx = -1;
   if (FORWARD) {
-    if (!costing_->Allowed(meta.edge, pred, tile, meta.edge_id, localtime, time_info.timezone_index,
-                           restriction_idx) ||
+    if (!costing_->Allowed(meta.edge, false, pred, tile, meta.edge_id, localtime,
```

Comment:
why always `false` ? If I remember correctly a destination edge can also be a connection edge

Reply:
from the other side it's a very specific case. even if we decline an edge on forward step we will use reverse search to make a connection. so, imho it can be always `false` but put some comment with explanation

Reply:
Added comments